A .html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+
+   <head>
+      <title>Hello Microverse!</title>
+   </head>
```

Comment:
- [ ] [OPTIONAL] It is one of the best practices in the industry as well as a convention to always include some meta tags to your page for accessibility reasons, SEO and correct page rendering. Something like 👇 

```
<head>
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Document</title>
</head>
```